runtime/Makefile

```diff
@@ -34,6 +36,9 @@ $(BCLIB): $(BCS)
 	sed -e '/define .*@_[Bb][a-zA-Z]/s/^define /define linkonce_odr /' -e '/target datalayout/ s/"$$/-ni:1"/' | \
 	$(LLVM_AS) >$@ 
 
+$(filter third-party/libbacktrace/%.o, $(OBJS)): third-party/libbacktrace/%.o: third-party/libbacktrace/%.c
```

Comment:
Make a definition for `$(addprefix third-party/libbacktrace/, $(LIBBACKTRACE_OBJS))` and use it here and in the definition of OBJS.

You don't need `third-part/libbacktrace/` in this rule.

Reply:
Fixed.